src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -17,15 +20,26 @@ interface PharmacyProps {
 }
 
 export function PharmacyCard(props: PharmacyProps) {
+  const { t, i18n } = useTranslation();
   const [shouldShowMap, setShouldShowMap] = useState(false);
+  function FormatDate(date: Date) {
+    // alert(i18n.language);
```

Comment:
We can remove the `alert`

Reply:
Removed. Does anyone know how to get the JavaScript debugger working in Visual Studio Code with yarn start (I'm using Firefox)?

Reply:
I would typical only use the VS Code debugger during tests... `yarn start` spins up a server to serve the app to a browser, which has its own runtime, and own debugger. Why wouldn't you use FF's devtools?